test/test-listing-guni.spec.ts

```diff
@@ -0,0 +1,277 @@
+    
+    // give GUNI to proposer
+    let gUniBalance = await guni0.balanceOf(GUNI_HOLDER);
+    await (
+      await guni0.transfer(
+        proposer.address,
+        gUniBalance.div(BigNumber.from("5"))
+      )
+    ).wait();
+
+    gUniBalance = await guni1.balanceOf(GUNI_HOLDER);
+    await (
+      await guni1.transfer(
+        proposer.address,
+        gUniBalance.div(BigNumber.from("5"))
+      )
+    ).wait();
+
+
+    // deploying the payload
+    await rawBRE.ethers.provider.send('evm_mine', [0]);
+
+    await rawBRE.deployments.deploy('AssetListingGUni', {
+      from: proposer.address,
+      gasLimit: 4000000,
+      gasPrice: BigNumber.from('75000000000'),
+      args: [],
+    });
+
+
+    proposal = await gov.getProposalsCount();
+
+    await rawBRE.run('create:proposal-new-asset:guni');
+
+    const payloadAddress = (
+      await rawBRE.deployments.get('AssetListingGUni')
+    ).address;
+    console.log(payloadAddress)
+
+    // voting, queuing proposals
+    await rawBRE.ethers.provider.send('evm_mine', [0]);
+
+    await (await gov.submitVote(proposal, true)).wait();
+    await advanceBlockTo((await latestBlock()) + VOTING_DURATION + 1);
+    await (await gov.queue(proposal)).wait();
+    let proposalState = await gov.getProposalState(proposal);
+    expect(proposalState).to.be.equal(5);
+    await increaseTime(86400 + 10);
+    snapshotId = await evmSnapshot();
+  });
+
+  it('Should list correctly an asset: borrow off, collateral on, stable borrow off', async () => {
+    await (await gov.execute(proposal)).wait();
+    
+    const proposalState = await gov.getProposalState(proposal);
+    expect(proposalState).to.be.equal(7);
+
+    const expectedConfig = {
+      reserveFactor: '1000',
+      reserved: '0',
+      stableRateEnabled: '0',
+      borrowingEnabled: '0',
+      reserveFrozen: '0',
+      reserveActive: '1',
+      decimals: '18',
+      liquidityBonus: '11500',
+      LiquidityThreshold: '7000',
+      LTV: '6000',
+    };
+
+    let reserveData = await pool.getReserveData(TOKEN0);
+    let poolData = parsePoolData(reserveData.configuration.data);
+    expect(poolData).to.be.eql(expectedConfig);
+
+    // preparing for tests on G-UNI DAI/USDC (guni0)
+    aGuni = (await ethers.getContractAt('IERC20', reserveData.aTokenAddress, proposer)) as IERC20;
+    stableDebt = (await ethers.getContractAt('IERC20', reserveData.stableDebtTokenAddress, proposer)) as IERC20;
+    variableDebt = (await ethers.getContractAt(
+      'IERC20',
+      reserveData.variableDebtTokenAddress,
+      proposer
+    )) as IERC20;
+    await (await guni0.connect(guniHolder).approve(pool.address, parseEther('200000'))).wait();
+
+    // GUNI deposit by guni holder
+    const depositedAmount0 = parseEther('3');
+    await (
+      await pool.connect(guniHolder).deposit(guni0.address, depositedAmount0, GUNI_HOLDER, 0)
+    ).wait();
+    expect(await aGuni.balanceOf(GUNI_HOLDER)).to.gte(depositedAmount0.sub(1));
```

Comment:
why not use `.to.eq`?

Reply:
haha agree! (was just copying pattern from FRAX pr)